controllers/csi/provisioner/reconciler.go

```diff
@@ -86,8 +78,9 @@ func (r *OneAgentProvisioner) SetupWithManager(mgr ctrl.Manager) error {
 var _ reconcile.Reconciler = &OneAgentProvisioner{}
 
 func (r *OneAgentProvisioner) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
-	rlog := log.WithValues("namespace", request.Namespace, "name", request.Name)
-	rlog.Info("Reconciling DynaKube")
+	log = log.WithValues("namespace", request.Namespace, "dynakube", request.Name)
+	defer resetLogger()
```

Comment:
we could explicitly log these values instead of having them on the package wide logger?